docs/end-to-end-demo-rpi4.md

```diff
@@ -48,7 +48,7 @@ This demo will demonstrate how to get Akri working on a **Raspberry Pi 4**, all
     ```
 1. Install Gstreamer main packages
     ```sh
-    sudo apt-get install -y libgstreamer1.0-0 gstreamer1.0-dev gstreamer1.0-tools \
+    sudo apt-get install -y libgstreamer1.0-0 libgstreamer1.0-dev gstreamer1.0-tools \
                             gstreamer1.0-doc gstreamer1.0-plugins-base gstreamer1.0-plugins-good \
                             gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly \
                             gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools \
```

Comment:
while we're cleaning ... it looks like `gstreamer1.0-doc` in in here twice.

Reply:
(and gstreamer1.0-tools)

Reply:
Curious, why do we need the doc package?

Reply:
We may not need it. I can investigate

Reply:
a lot ended up not being needed in order to mock the videos, including the `libgstreamer1.0-dev` which prompted this PR